CHANGELOG.md

```diff
@@ -13,10 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add support to remote write to Cortex
 - Added recording rules
 - Add node affinity to prefer not scheduling on master nodes
+- Added `pipeline` tag to _Hearbeat_ alert to be able to see if it affects
+  a stable or testing installation at first glance
+- Added initial support for `remote_write` that will eventually be used for
+  writing to Cortex
```

Comment:
changelog is not subject to 80 character line wrap AFAIK. I'm not strong on this.

Reply:
Sure, it's a habit for me by now, I like the 80 char limit as it makes reading in plain-text form easier. I'll go with this for now but I'll try to relax this a bit in the future 😉